app/views/facility_account_orders/index.html.haml

```diff
@@ -32,16 +32,14 @@
           %td= order_detail.ordered_at
           %td
             %ul
+              - reservation = order_detail.reservation
               %li= order_detail.description_as_html
-              - if order_detail.reservation.present?
-                - if order_detail.reservation_admin_editable?
-                  %li.indented
-                    = link_to order_detail.reservation,
-                      order_detail.edit_reservation_path
-                - else
-                  %li.indented
-                    = link_to order_detail.reservation,
-                      order_detail.show_reservation_path
+              - if reservation.present?
```

Comment:
What would you think of replacing this whole chunk with the `order_detail_cell` partial? It would address the link problem as well as the extra list item dots.